helm/trivy/README.md

```diff
@@ -54,36 +54,37 @@ The command removes all the Kubernetes components associated with the chart and
-| `trivy.registryPassword`              | The password used to log in at dockerhub. More info: https://aquasecurity.github.io/trivy/dev/advanced/private-registries/docker-hub/ |      |
-| `trivy.registryCredentialsExistingSecret` | Name of Secret containing dockerhub credentials. Alternative to the 2 parameters above, has precedence if set.                    |      |
-| `trivy.serviceAccount.annotations`        | Additional annotations to add to the Kubernetes service account resource |     |
-| `trivy.skipUpdate`                    | The flag to enable or disable Trivy DB downloads from GitHub            | `false`        |
-| `trivy.dbRepository`                  | OCI repository to retrieve the trivy vulnerability database from        | `ghcr.io/aquasecurity/trivy-db`        |
-| `trivy.cache.redis.enabled`           | Enable Redis as caching backend                                         | `false` |
-| `trivy.cache.redis.url`               | Specify redis connection url, e.g. redis://redis.redis.svc:6379         | `` |
-| `trivy.serverToken`                   | The token to authenticate Trivy client with Trivy server                | `` |
-| `trivy.podAnnotations`                | Annotations for pods created by statefulset                             | `{}` |
-| `service.name`                        | If specified, the name used for the Trivy service                       |     |
-| `service.type`                        | Kubernetes service type                                                 | `ClusterIP` |
-| `service.port`                        | Kubernetes service port                                                 | `4954`      |
-| `httpProxy`                           | The URL of the HTTP proxy server                                        |     |
-| `httpsProxy`                          | The URL of the HTTPS proxy server                                       |     |
-| `noProxy`                             | The URLs that the proxy settings do not apply to                        |     |
-| `nodeSelector`                        | Node labels for pod assignment                                              |     |
-| `affinity`                            | Affinity settings for pod assignment                                              |     |
-| `tolerations`                         | Tolerations for pod assignment                                              |     |
-| `podAnnotations`                      | Annotations for pods created by statefulset                             | `{}` |
+|                 Parameter             | Description                                                                                                                           |    Default   |
+|---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|--------------|
```

Comment:
@cebidhem  I don´t see any difference between this files, can you remove the extra spaces? The README is identical of the original version.

Reply:
Done. Indeed you'll better see my addition now!